student.rb

```diff
@@ -1,9 +1,21 @@
 require_relative 'person'
 
 class Student < Person
+  attr_accessor :classroom
+
   def initialize(age, classroom, parent_permission: true, name: 'Unknown')
     super(age, parent_permission: parent_permission, name: name)
-    @classroom = classroom
+    @classroom = nil
+    self.classroom = classroom
+  end
```

Comment:
- [ ] It would be nice to separate this implementation from the initialize method and create a separate method that serves as the 'belongs-to' side (a student belongs to a classroom).
- The code below will help achieve that: 👍🏽
```ruby
def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```